integ-test/build.gradle

```diff
@@ -91,7 +91,7 @@ dependencies {
     testRuntimeOnly('org.junit.jupiter:junit-jupiter-engine:5.6.2')
 
     testImplementation group: 'com.h2database', name: 'h2', version: '2.1.214'
-    testImplementation group: 'org.xerial', name: 'sqlite-jdbc', version: '3.28.0'
+    testImplementation group: 'org.xerial', name: 'sqlite-jdbc', version: '3.32.3.3'
```

Comment:
When I first cloned the repository I would have a few integration tests fail and Andrew figured out that it was because that version of sqlite-jdbc doesn't work on M1 Macs, I do not know if this is some that I should commit or not.